README.md

```diff
@@ -3,6 +3,9 @@
 | [![Build status][3.0 image]][3.0] | [![Build status][2.12 image]][2.12] | [![Build status][2.11 image]][2.11] |
 | [![Coverage Status][3.0 coverage image]][3.0 coverage]| [![Coverage Status][2.12 coverage image]][2.12 coverage] | [![Coverage Status][2.11 coverage image]][2.11 coverage]  |
 
+
+[![SWUbanner](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://www.doctrine-project.org/stop-war.html)
```

Comment:
We shouldn't user their banner if we link to our own page.

Reply:
I saw nothing wrong to that, but well fixed.